.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,51 @@
+name: Error Reference Publisher
+on:
+  push:
+    branches:
+      - 'err-ref-ci'
+
+jobs:
+  error-ref:
+    name: Error codes utility
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . analyze --skip-dirs meshery
+
+      # to update errorutil* files in meshkit repo
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_user_name: l5io
+          commit_user_email: ci@layer5.io
+          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit_options: '--signoff'
+          commit_message: "run error codes utility"
+
+      - name: Checkout meshery
```

Comment:
Does it checkout `master` branch by default? We would want to make sure that happens